sql/02-bde_control_functions.sql.in

```diff
@@ -1953,12 +1953,6 @@ BEGIN
         v_task := 'Deleting data from current version of the table ' || v_bdetable;
         RAISE NOTICE '%', v_task;
 
-        IF table_version.ver_is_table_versioned(v_bde_schema, p_table_name)
```

Comment:
@strk , why you are deleting this ?

Reply:
The `bde_admin` role does NOT have permissions to disable versioning on tables it doesn't own. BDE tables are owned by `bde_dba` who is then the only one with rights to disable versioning on them.

That code path is only hit when doing a non-incremental LEVEL0 update. Versioning used to be disabled on the target table because the table would have been _replaced_ with a new version, so for speed reasons I guess ?

Now that we TRUNCATE the table instead of doing a DROP/REPLACE there is no real point in disabling versioning (unless https://github.com/linz/postgresql-tableversion/issues/204 will strike one day).

Note that in production we will NOT have versioning enabled at the time we do a non-incremental LEVEL0 update...

Reply:
Note that the above is also explained in the commit log: https://github.com/linz/linz-bde-uploader/pull/249/commits/f3382444b92d1fc0fab6d4a5c2ef88f8faebe324

Reply:
Thanks for explanation.